modules/install-sno-requirements-for-installing-on-a-single-node.adoc

```diff
@@ -25,12 +25,14 @@ Installing {product-title} on a single node alleviates some of the requirements
 +
 [NOTE]
 ====
-One vCPU is equivalent to one physical core when simultaneous multithreading (SMT), or hyperthreading, is not enabled. When enabled, use the following formula to calculate the corresponding ratio:
-
+* One vCPU is equivalent to one physical core when simultaneous multithreading (SMT), or hyperthreading, is not enabled. When enabled, use the following formula to calculate the corresponding ratio:
++
 (threads per core × cores) × sockets = vCPUs
+
+* Adding operators during the installation process might increase the minimum resource requirements.
```

Comment:
See OpenShift documentation guidelines for [Operator capitalization](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#operator-capitalization).
```suggestion
* Adding Operators during the installation process might increase the minimum resource requirements.
```